i18n/en.json

```diff
@@ -6612,7 +6084,7 @@
   },
   {
     "id": "web.get_access_token.internal_saving.app_error",
-    "translation": ""
+    "translation": "We couldn't update the user access data."
```

Comment:
Is this the token for the user to log in, or the personal access token? (same for the above two)

Reply:
Actually, it is in the OAuth code and looks like it is related to the update of the session token.